File: node/sdk/controllers/subscribeController.js
import express from "express";

const router = express.Router();

export const subscribeController = router.get("/", (_req, res) => {
    try {
        res.json([
            {
                pubsubname: "pubsub",
                topic: "orders",
                route: "orders"
            }
        ]);
    } catch (error) {
        console.error(error)
        res.json({ "err": error })
    }
})

File: dotnet/sdk/Controllers/SubscribeController.cs
using Microsoft.AspNetCore.Mvc;
using Dapr;

namespace http.Controllers;

[ApiController]
public class SubscribeController : ControllerBase
{
    [Topic("pubsub", "orders")]
    [HttpPost("/orders")]
    public string Subscribe([FromBody] dynamic content)
    {
        string order = "Order recieved with information: " + content.ToString();
        Console.WriteLine(order);
        return order;
    }
}